packages/sharing-smart-contract/contracts/modules/Collection.sol

```diff
@@ -39,6 +39,14 @@ contract Collection is ERC721Burnable, ERC721Receiver {
         _;
     }
 
+    modifier onlyProtectedDataOwnByCollection(uint256 _collectionId, address _protectedData) {
```

Comment:
```suggestion
    modifier onlyProtectedDataOwnedByCollection(uint256 _collectionId, address _protectedData) {
```

Reply:
Fix in PR-120 by onlyProtectedDataInCollection